cdap-docs/developers-manual/source/building-blocks/namespaces.rst

```diff
@@ -37,12 +38,16 @@ Namespace IDs are composed from a limited set of characters; they are restricted
 letters (a-z, A-Z), digits (0-9), hyphens (-), and underscores (_). There is no size limit
 on the length of a namespace ID nor on the number of namespaces.
 
-The namespace IDs ``default`` and ``system`` are reserved. The ``default``
+The namespace IDs ``cdap``, ``default``, and ``system`` are reserved. The ``default``
 namespace, however, can be used by anyone, though like all reserved namespaces, it cannot
 be deleted.
 
-Namespaces: Independent and Non-hierarchal
-------------------------------------------
+Once created, these namespace properties (ID, display name, description) cannot be altered. 
```

Comment:
Begs another point though - we need to add documentation for a new API. A namespace can have properties now. I guess we could do that in a later PR.